### PR TITLE
[EventHubs] `az eventhubs eventhub`: Migrating EventHub entity and Schema registry cmdlets to aaz

### PR DESCRIPTION
**Pull request by damodaravadhani**
**Related command**
Command group: `az eventhubs eventhub`

**Description**
Migrating all cmdlets from the above command group to aaz.

**Testing Guide**
Refer tests

**History Notes**

[Component Name 1] BREAKING CHANGE: `az command a`: Make some customer-facing breaking change
[Component Name 2] `az command b`: Add some customer-facing feature

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [X] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [X] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [X] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
EventHubs

**Comment by damodaravadhani** [comment]
@evelyn-ys, would you be able to help with a review when you find time ?

**Comment by evelyn-ys** [comment]
@damodaravadhani Please fix all CI checks

**Comment by damodaravadhani** [comment]
@evelyn-ys, I couldn't understand why Integration Test pipelines are failing, can you help ?

**Comment by evelyn-ys** [comment]
@kairu-ms Can you help figure out why CI fails with after migration to aaz:
```
Error loading command module 'eventhubs': No module named 'azure.cli.command_modules.eventhubs.operations'
```

**Comment by kairu-ms** [comment]
> @kairu-ms Can you help figure out why CI fails after migration to aaz:
> 
> ```
> Error loading command module 'eventhubs': No module named 'azure.cli.command_modules.eventhubs.operations'
> ```

The `__init__.py` file missed

**Comment by damodaravadhani** [comment]
> > @kairu-ms Can you help figure out why CI fails after migration to aaz:
> > ```
> > Error loading command module 'eventhubs': No module named 'azure.cli.command_modules.eventhubs.operations'
> > ```
> 
> The `__init__.py` file missed

Thanks for the quick update @kairu-ms , there seem to be __init__.py in almost all the directories and I have pushed all of them from my local. Can you help me figure out __init__.py from which directory is missing ?

**Comment by damodaravadhani** [comment]
> > > @kairu-ms Can you help figure out why CI fails after migration to aaz:
> > > ```
> > > Error loading command module 'eventhubs': No module named 'azure.cli.command_modules.eventhubs.operations'
> > > ```
> > 
> > 
> > The `__init__.py` file missed
> 
> Thanks for the quick update @kairu-ms , there seem to be **init**.py in almost all the directories and I have pushed all of them from my local. Can you help me figure out **init**.py from which directory is missing ?

Oh I see, it is not added in the operations folder.

**Comment by damodaravadhani** [comment]
@kairu-ms, should we also raise a corresponding PR to aaz-dev repo also ?

**Comment by kairu-ms** [comment]
> @kairu-ms, should we also raise a corresponding PR to aaz-dev repo also ?

Yes, please create a PR in [aaz](https://github.com/Azure/aaz) repo

**Comment by damodaravadhani** [comment]
@evelyn-ys, can we merge this if there aren't additional concerns ?

File: src/azure-cli/azure/cli/command_modules/eventhubs/tests/latest/test_cluster_commands.py [hunk 1]
```diff
@@ -42,24 +42,24 @@ def test_eh_cluster(self):
...
         # Create Namespace in cluster
-        self.cmd('eventhubs namespace create --resource-group {rg} --name {namespacename} --location {loc} --tags {tags} --sku {sku} --cluster-arm-id {clusterid}',
-                 checks=[self.check('sku.name', self.kwargs['sku']),
-                         self.check('clusterArmId', self.kwargs['clusterid'])])
+        self.cmd('eventhubs namespace create --resource-group {rg} --name {namespacename} --location {loc} --tags {tags} --sku {sku} --cluster-arm-id {clusterid}')
 
         # Get namespaces created in the cluster
-        listnsclusterresult = self.cmd('eventhubs cluster namespace list --resource-group {rg} --name {clustername}').output
-        self.assertGreater(len(listnsclusterresult), 0)
+        listnsclusterresult = self.cmd('eventhubs cluster namespace list --resource-group {rg} --name {clustername}').get_output_in_json()
```
Comment (evelyn-ys) [inline src/azure-cli/azure/cli/command_modules/eventhubs/tests/latest/test_cluster_commands.py:54]: Does this command's output change from string to json?

File: src/azure-cli/azure/cli/command_modules/eventhubs/tests/latest/test_cluster_commands.py [hunk 2]
```diff
@@ -42,24 +42,24 @@ def test_eh_cluster(self):
...
+        self.assertEqual(len(listnsclusterresult), 1)
 
         # update cluster
         self.cmd('eventhubs cluster update --resource-group {rg} --name {clustername} --tags tag2=value2',
                  checks=[self.check('tags', {'tag2': 'value2'})])
 
         # Get cluster created in the resourcegroup
-        listclusterresult = self.cmd('eventhubs cluster list --resource-group {rg}').output
+        listclusterresult = self.cmd('eventhubs cluster list --resource-group {rg}').get_output_in_json()
```
Comment (evelyn-ys) [inline src/azure-cli/azure/cli/command_modules/eventhubs/tests/latest/test_cluster_commands.py:62]: Same here

File: src/azure-cli/azure/cli/command_modules/eventhubs/aaz/latest/eventhubs/eventhub/_create.py [hunk 3]
```diff
@@ -0,0 +1,400 @@
...
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "eventhubs eventhub create",
+)
+class Create(AAZCommand):
+    """Create a new Event Hub as a nested resource within a Namespace.
+    """
```
Comment (kairu-ms) [inline src/azure-cli/azure/cli/command_modules/eventhubs/aaz/latest/eventhubs/eventhub/_create.py:19]: please provide examples for create and update commands.

**Pull request closed without merging**